physionet-django/console/utility.py

```diff
@@ -1,6 +1,9 @@
 from os import walk, chdir, listdir, path
+import pdb
```

Comment:
is this intentional?

Reply:
import pdb is in most files, and its quite useful to have imported even if its not used at the moment.